scripts/components/security/build.sh

```diff
@@ -59,22 +60,16 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
-OPENSEARCH_RELEASE=$VERSION
 [[ "$SNAPSHOT" == "true" ]] && VERSION=$VERSION-SNAPSHOT
```

Comment:
The properties file is still including the `-SNAPSHOT` on the version, trying to figure out what isn't working there, this shouldn't be merged until that has been resolved